src/java/org/jivesoftware/LoginDialog.java

```diff
@@ -277,11 +239,22 @@ protected XMPPTCPConnectionConfiguration retrieveConnectionConfiguration() {
         }
 
         final XMPPTCPConnectionConfiguration.Builder builder = XMPPTCPConnectionConfiguration.builder()
-                .setUsernameAndPassword( "username", "password" )
+                .setUsernameAndPassword( loginUsername, loginPassword )
                 .setServiceName( loginServer )
                 .setPort( port )
                 .setSendPresence( false )
                 .setCompressionEnabled( localPref.isCompressionEnabled() );
+        try {
+            TLSUtils.acceptAllCertificates(builder);
+        } catch (NoSuchAlgorithmException e) {
+            e.printStackTrace();
```

Comment:
Instead of `e.printStackTrace();`  please do `Log.warn( "Unable to create configuration.", e );` which will make sure that the error ends up in the logfiles (As a rule of thumb, never do `e.printStackTrace()` ).